Source/SwiftLintFramework/Rules/Idiomatic/ToggleBoolRule.swift

```diff
@@ -81,19 +80,18 @@ private extension ToggleBoolRule {
 
             correctionPositions.append(node.positionAfterSkippingLeadingTrivia)
 
-            let functionCall = FunctionCallExprSyntax { functionCall in
-                functionCall.useCalledExpression(
-                    ExprSyntax(
-                        MemberAccessExprSyntax { memberAccess in
-                            memberAccess.useBase(node.first!.withoutTrivia())
-                            memberAccess.useDot(.period)
-                            memberAccess.useName(.identifier("toggle"))
-                        }
+            let functionCall = FunctionCallExprSyntax(
```

Comment:
You could probably do this with:

```swift
let functionCall: ExprSyntax = "\(node.first!.withoutTrivia()).toggle()"
```

Reply:
Nice one!

Reply:
Thanks for the suggestion, that's the same one I had demonstrated here: https://github.com/apple/swift-syntax/pull/782#issuecomment-1244064686